package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "ajax-service",
+  "version": "1.0.0",
+  "description": "ajax-service is used as a promise-based abstraction of XMLHttpRequest.",
+  "main": "dist/ajax.js",
+  "scripts": {
+    "build": "npm run build:lint && npm run build:compile",
+    "build:lint": "tslint src/*",
+    "build:compile": "tsc"
+  },
```

Comment:
@Izzmo @samiam236 I am using strictly NPM scripts to help with the simplicity for these repos.  `npm run build` will now lint and compile everything correctly.